lib/cocoapods/installer.rb

```diff
@@ -71,6 +71,7 @@ def initialize(sandbox, podfile, lockfile = nil)
       @lockfile = lockfile
 
       @use_default_plugins = true
+      @has_dependencies = true
```

Comment:
While we're at it, can we also set `repo_update` to `true` here, since that's the default? i'm not sure how I missed that in the other PR. Thanks!